src/components/fields/hidden-field/hidden-field.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react";
+import * as Yup from "yup";
+import { IGenericFieldProps } from "..";
+import { useValidationConfig } from "../../../utils/hooks";
+import { IHiddenFieldSchema } from "./types";
+
+export const HiddenField = (props: IGenericFieldProps<IHiddenFieldSchema>) => {
+	// =============================================================================
+	// CONST, STATE, REFS
+	// =============================================================================
+	const {
+		id,
+		schema: { showIf: _showIf, uiType: _uiType, validation, ...otherSchema },
+		...otherProps
+	} = props;
+	const { setFieldValidationConfig } = useValidationConfig();
+
+	// =============================================================================
+	// EFFECTS
+	// =============================================================================
+	useEffect(() => {
+		setFieldValidationConfig(id, Yup.string(), validation);
```

Comment:
something to consider as a future enhancement is the ability to handle other types. a pain point I've encountered is having to cast boolean to string.

Reply:
oh yeah thanks for the suggestion, I will introduce a new validation rule for hidden field to validate as other data types